src/components/ec-blog-item/ec-blog-item.spec.js

```diff
@@ -0,0 +1,28 @@
+import { mount } from '@vue/test-utils';
+import EcBlogItem from './ec-blog-item.vue';
+
+const post =
+  {
+    title: 'title',
+    shortDescription: 'shortDescription',
+    author: 'author',
+    category: 'category',
+  };
```

Comment:
Can we make these texts more random. And by that I mean set them to values that are not the same as prop names. If I accidentally change `<p>{{ author }}</p>` to `<p>author</p>` in your component, this test will still be green. Make it fail in that situation. You can just set `title` to be `My Blog Post`, `shortDescription` to `Lorem ipsum ...`, `author` to be `Random Author`, etc.